src/Uno.UWP/UI/Notifications/BadgeUpdater.macOS.cs

```diff
@@ -1,22 +1,15 @@
-#if __MACOS__
+#nullable enable
```

Comment:
I think I read a comment somewhere that public APIs shouldn't enable nullable reference types since UWP isn't yet annotated.

Reply:
I have enabled nullables on `FocusManager` before, so I hope it is not a problem. It is not a potentially breaking change, so as long as the nullables match the UWP expectations, it should be ok

Reply:
I'll let @jeromelaban check that :)